apps/settings/js/panels/feedback_send/feedback_send.js

```diff
@@ -106,13 +110,14 @@ define(function(require) {
         currentSetting['deviceinfo.hardware'];
       this._sendData.locale =
         currentSetting['language.current'];
+      this._sendData.happy = this._isHappy();
```

Comment:
I can't comment on other parts of this file which is irritating.

Do you also need to add happy handling to get/set `_inputData`?